pkg/cmd/cli/cmd/create.go

```diff
@@ -67,160 +65,16 @@ func CmdCreate(f *clientcmd.Factory, reader io.Reader, out io.Writer) *cobra.Com
 }
 
 func (o *CmdOptions) RunCreate(out io.Writer, cmd *cobra.Command, args []string) error {
-	// pre spark 2, the name the master calls itself must match
-	// the name the workers use and the service name created
-	masterhost := o.Name
-
-	// Create the master deployment config
-	dcc := o.Client.DeploymentConfigs(o.Project)
-	masterdc := sparkMaster(o)
-
-	// Create the services that will be associated with the master pod
-	// They will be created with selectors based on the pod labels
-	mastersv, _ := service(masterhost,
-		masterdc.FindPort(masterPortName),
-		o.Name, masterType,
-		masterdc.GetPodTemplateSpecLabels())
-
-	websv, _ := service(masterhost+"-ui",
-		masterdc.FindPort(webPortName),
-		o.Name, webuiType,
-		masterdc.GetPodTemplateSpecLabels())
-
-	// Create the worker deployment config
-	//masterurl := sparkMasterURL(masterhost, &masterp.ServicePort)
-	workerdc := sparkWorker(o)
-
-	// Launch all of the objects
-	_, err := dcc.Create(&masterdc.DeploymentConfig)
-	if err != nil {
-		return ErrorString(mDepConfigMsg, err)
-	}
-	_, err = dcc.Create(&workerdc.DeploymentConfig)
-	if err != nil {
-		// Since we created the master deployment config, try to clean up
-		deleteCluster(o)
-		return ErrorString(wDepConfigMsg, err)
-	}
-
-	// If we've gotten this far, then likely the cluster naming is not in conflict so
-	// assume at this point that we should use a 500 error code
-	sc := o.KClient.Services(o.Project)
-	_, err = sc.Create(&mastersv.Service)
+	config := clusterconfigs.ClusterConfig{}
+	config.WorkerCount = o.WorkerCount
```

Comment:
There is a wrinkle, however. Have to think about the scale case -- default results in an error, since it tries to scale to zero rather than 1. There might be an issue in oshinko-core, though, so I would call this a non-issue for the moment.  One could argue that scale without a count dropping to "1" is not something you want anyway.  So maybe the arg shold be required.

Don't see this is an issue to delay merge, though.